src/components/Check/Check.scss

```diff
@@ -7,33 +7,63 @@
 // --------------------------------------------------
 // Check styles
 
+$checkBoxHeight: 20px;
+
 .ms-Check {
   display: inline-block;
   cursor: default;
-  line-height: 0;
   vertical-align: top;
+  width: $checkBoxHeight;
+  height: $checkBoxHeight;
+  text-align: center;
 
-  &.is-checked {
+  .ms-Check-background {
+    &:before {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      display: inline-block;
+      margin: auto;
+      height: $checkBoxHeight - 2px;
+      width: $checkBoxHeight - 2px;
+      background: $ms-color-white;
+      border: 1px solid $ms-color-neutralTertiaryAlt;
+      border-radius: 50%;
+      content: ' ';
+    }
 
-    .ms-Check-circle {
-      fill: $ms-color-themePrimary;
-      stroke: $ms-color-white;
-      stroke-width: 1px;
+  }
+
+  &.is-checked {
+    .ms-Check-background {
+      &:before {
+        background: $ms-color-themePrimary;
+        border: 1px solid $ms-color-white;
+        z-index: $ms-zIndex-1;
```

Comment:
Double check if z-index is needed here and in .ms-Check-check. We try to avoid z-index as they have created problems in past :)